pkg/operator/operator.go

```diff
@@ -78,6 +82,7 @@ func (c OpenShiftControllerManagerOperator) sync() error {
 		return err
 	}
 	switch operatorConfig.Spec.ManagementState {
+	// if we decide to not allow this operator to be set unmanaged, then remove the management.SetOperatorAlwaysManaged() call in NewOpenShiftControllerManagerOperator()
 	case operatorapiv1.Unmanaged:
```

Comment:
Is there a chance we do *NOT* want this operator to be unmanaged/removed @bparees @adambkaplan ?

Reply:
we do not want it to be removed for sure. Doing so will apparently break lots of stuff because things like the SCC controller run inside this pod and if it's not there, pods can't be admitted.

/cc @deads2k 

Presumably we also don't want to allow unmanaged, for similar reasons, but that's less clear to me since it might be needed/useful for manual intervention on a cluster.

Reply:
@gabemontero my vote is that this operator should _never_ be removed. It's also hard for me to envision a situation where a customer would need to set this to Unmanaged, but keep other control plane components managed.
@bparees or @derekwaynecarr may have a different take.